File: allSpice/Models/Recipe.cs
namespace allSpice.Models
{
  public class Recipe
  {
    public int Id { get; set; }
    public string title { get; set; }
    public string instructions { get; set; }
    public string category { get; set; }
    public string img { get; set; }
    public string creatorId { get; set; }
    public Account creator { get; set; }
  }

  public class SubscriberRecipe : Recipe
  {
    public int SubscriptionId { get; set; }
  }
}

File: allSpice.client/src/models/Recipe.js
export class Recipe {
    constructor(data) {
        this.id = data.id
        this.title = data.title
        this.instructions = data.instructions
        this.category = data.category
        this.img = data.img
        this.creatorId = data.creatorId
        this.creator = data.creator
    }
}

export class SubscriptionRecipe extends Recipe{
    constructor(data){
        super(data)
        this.subscriptionId = data.subscriptionId
    }
}